.gitignore

```diff
@@ -11,6 +11,10 @@
 !/bin
 /bin/*
 !/bin/brew
+!/share
+/share/*
+!/share/doc
+/share/doc/*
```

Comment:
Why this is necessary? If it's indeed required, you will have to update corresponding  logic in uninstall script: https://github.com/Homebrew/install/blob/master/uninstall#L107-L118